src/promptflow-evals/promptflow/evals/evaluate/_eval_run.py

```diff
@@ -304,6 +320,17 @@ def _log_warning(self, failed_op: str, response: requests.Response) -> None:
             f"{response.text=}."
         )
 
+    def _raise_not_started_nay_be(self) -> None:
+        """
+        Raise value error if the run was not started.
+
+        :raises: ValueError
+        """
+        if self._status == RunStatus.NOT_STARTED:
+            raise ValueError(

```

Comment:
When is this error raised ? And is customer expected to do something to resolve it ?

Reply:
This check is added for the integrity, this class is only used internally and if this exception has raised, it means the bug. It is essentially done to fail the unit/e2e tests if that has somehow happened.

Reply:
In this case should be marked as broken, by our code the run should never be in Non-started state. This exception is added for code integrity, in case we will occasionally change this logic.

Reply:
Fixed, made it a warning.